Sources/CreateAPI/Generate.swift

```diff
@@ -142,6 +146,11 @@ struct Generate: ParsableCommand {
         return GenerateOptions(options)
     }
     
+    private func applyTemplateToEntityIncludeAndExclude(options: GenerateOptions) {
```

Comment:
I think this is not smart but needed when you use entities.include or entities.exclude. Is there any other good solutions for this?